### DISCUSSION TITLE
NEW to Computer Science and coding

### DISCUSSION THREAD

**Discussion by Artyieee** [microsoft/Web-Dev-For-Beginners#1097] [General]
Hello, I'm at level 0 and don't have any knowledge about CS however I want to learn a language so I started with github.
Can anyone recommend if I should watch videos on EDx, Coursera etc to enhance my knowledge of the whole thing or go for books?
( I would prefer videos)
 Please share your suggestions on how to get started with this whole thing I really want to learn and to code.

**Comment by ghost** [comment]
I would check out freeCodeCamp.org, there's a ton of courses available to teach you complete basics of front end, back end & full-stack development. I also recommend watching the videos on YouTube that go along with the courses. Upon completion you'll receive certificates to validate your learning progress. The Odin project is another great example of where to find a learning path. Depending on your goals & what system you're working on, VS Code & Atom are great Editors. I also recommend checking out Replit, there's tutorials on a variety of languages & you can now access your Repls via mobile by downloading the Repit mobile app that's available on android & iOS.

**Comment by E-phraim** [comment]
hey, we can be learning partners. check out these resources - 
- https://www.freecodecamp.org/news/how-i-went-from-newbie-to-software-engineer-in-9-months-while-working-full-time-460bd8485847/
- https://pll.harvard.edu/course/cs50-introduction-computer-science
